src/main/java/usecase/alter_entry/AlterEntryUseCase.java

```diff
@@ -0,0 +1,56 @@
+package usecase.alter_entry;
+
+import billgates.database.QueryEntryData;
+import billgates.interface_adapters.DatabaseGateway;
+
+import java.time.ZonedDateTime;
+
+public class AlterEntryUseCase implements AlterEntryInputPort {
+    private final DatabaseGateway gateway;
+
+    public AlterEntryUseCase(DatabaseGateway gateway) {
+        this.gateway = gateway;
+    }
+
+
+    @Override
+    public void alterEntry(int billId, int entry_id, Object new_value, String alter_column) {
```

Comment:
As I said in WeChat, we may change this accordingly with Ray's update.

Sorry for the inconvenience.

Reply:
OK